pkg/infoschema/infoschema_v2.go

```diff
@@ -95,6 +95,17 @@ type Data struct {
 
 	// For information_schema/metrics_schema/performance_schema etc
 	specials map[string]*schemaTables
+
+	// pid2tid is used by FindTableInfoByPartitionID, it stores {partitionID, schemaVersion} => table ID
+	// Need full data in memory!
```

Comment:
When we first init infoschemaV2, do we need to insert all pid2tid item?

Reply:
Yes, add() will do it.
https://github.com/pingcap/tidb/pull/52479/files#diff-d82e5bb6499a8d8c00f3d55ec95bccda3fab6d4cc45529cecf06552bfa973b1bR169